db/models/issue.model.js

```diff
@@ -147,6 +147,11 @@ class Issue extends Model {
       sourceKey: "id",
       as: "payments"
     });
+    this.belongsTo(models.repositories, {
```

Comment:
repositories should cease to exist, since we don't only rely on github for bounty creation